app/pages/Recovery/util.ts

```diff
@@ -103,7 +103,9 @@ function extractCredentialIndexAndPolicy(
         accountInfo.accountCredentials
     ).find(
         ([, cred]) =>
-            (cred.value.contents.credId || cred.value.contents.regId) === credId
+            (cred.value.type === 'initial'
```

Comment:
We do this quite a bit. Perhaps a helper function taking a credential and returning the `credId` would be nice to have.